src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+
+  width: $card-width;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__img {
+    margin-bottom: 40px;
+    max-height: 134px;
+
+    object-fit: contain;
+  }
+
+  &__title {
+    @extend %basic-text;
+
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    @extend %small-text;
+
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    color: $color-secondary;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    @extend %basic-text;
+
+    font-weight: 400;
+    color: $color-secondary;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &__buy {
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    padding: 11px 0;
+
+    border: 1px solid transparent;
+    border-radius: 5px;
+    background: $color-accent-blue;
+
+    &:hover {
+      border-color: $color-accent-blue;
+      border-radius: 5px;
+      background: #fff;
```

You create variable for colors, but not for all 🥺 
```suggestion
      background: $c-white;
```